python-template/{{cookiecutter.placeholder_repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,8 @@
-**Description:** Describe in a couple of sentences what this PR adds
```

Comment:
I'm not sure if there is a relevant OEP, or if we should just create an ADR in this repo, but the cookiecutter is making default decisions across repos, so it would be worth vetting changes like this across a wider audience.

Reply:
I've made an identical PR for event-bus-kafka so we can try it out there: https://github.com/openedx/event-bus-kafka/pull/29

Reply:
I don't think an OEP or ADR is necessary because I think this change and its commit message are self-documenting. That said, it would be fair to circulate the PR around a few public Slack channels in order to expose it to a wider audience.